dev-libs/tvision/tvision-2.2.3.ebuild

```diff
@@ -56,6 +56,8 @@ src_configure() {
 	myconf+=(
 		--fhs
 		--prefix="${EPREFIX}/usr"
+		# avoid stripping
+		--with-debug
```

Comment:
Checked this doesn't do anything irritating like append -g* etc?

Reply:
These two lines get added to the Makefile:
```
$stripDebug=($OSf eq 'Darwin') ? '-S' : '--strip-debug';
$rep.="\tstrip $stripDebug \$(libdir)/$ver\n" unless $conf{'debugInfo'} eq 'yes';
```
From my POV it only calls `strip --strip-debug` with this option enabled. I also didn't notice any unusal compiler flags added during my test. I had enabled in older ebuilds with the USE=debug flag, but noticed this actually didn't add any debugging information, so removed it. But I didn't recognized that it's enabling stripping.
The `$conf{'debugInfo'}` is set to yes with the `--with-debug` option.

Reply:
For some context: https://github.com/set-soft/tvision/blob/master/tvision/config.pl#L1281